### PR TITLE
creating validation table for final ver

### PR DESCRIPTION
**Pull request by smuinsar**

### CODE REVIEW

File: validation_data/CSLC_Create_Validation_Table.ipynb [hunk 1]
```diff
@@ -74,25 +74,27 @@
     "DATA_ROOT = 'science/SENTINEL1'\n",
```
Comment (gracebato) [inline validation_data/CSLC_Create_Validation_Table.ipynb:74]: @smuinsar <p>You can actually delete/comment this. You only need to load the <code>validation_bursts_target_v{version_num}.csv which is already in the above cell.</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OPERA-Cal-Val/calval-CSLC/pull/22/discussion/#comment-1340798969'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (smuinsar) [reply validation_data/CSLC_Create_Validation_Table.ipynb:74]: csv file can be read into burst list. 
I added more corrections.
In run1_ALE_toggle_bursts_parallel.py, new CR data in Rosamond does not need solid earth tide correction.
Oversampling factor can be 32 or 64. 32 should work well with OPERA CSLC. Too large oversampling factor results in too large memory consumption and processing speed. See _Gisinger's 2021 paper:_ [10.1109/TGRS.2019.2961248](https://doi.org/10.1109/TGRS.2019.2961248)

**Comment by gracebato** [comment]
If you remember back in Spring, we already tested the oversampling factor number. It is indeed known that it will take longer to produce ALE results and it consumes memory. But we only have a couple of data to do this. In summary, it should not be an issue. 128x also produced the most accurate one. I will not change this anymore.

I already removed the solid earth for ALE after discussing with Ron M. but when I rechecked I failed to commit the changes. It did not impact the ALE results.

**Comment by gracebato** [comment]
@smuinsar please revert the oversampling to 128 and remove the `burstId_list` to reduce clutter in the code and as this is also redundant in the notebook. Thanks,

**Pull request merged**